lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlBasic.pm

```diff
@@ -0,0 +1,96 @@
+package Genome::Model::Tools::Analysis::DumpIgvXmlBasic;
+
+use strict;
+use warnings;
+
+use Genome;
+use Command;
+use IO::File;
+use Cwd qw(abs_path);
+use URI;
+use XML::Writer;
+
+class Genome::Model::Tools::Analysis::DumpIgvXmlBasic {
+    is => 'Command',
+    has => [
+        resource_files => {
+            type => 'HASH',
+            doc => 'A keyed list of files to display in the IGV session. Each key is a path to a file and the corresponding value is that label that should be displayed in the IGV session for that file.',
+        },
+        output_file => {
+            type => 'String',
+            is_optional => 0,
+            doc => 'Output XML file',
+        },
+        reference_name => {
+            type => 'String',
+            is_optional => 0,
+            doc => 'the name of the reference (in IGV) that the bams are aligned to. E.g. "b37" for build 37 or "reference"for our internal build36',
+        },
+    ]
+};
+
+sub help_brief {
+    "Makes an IGV session for review. Supports as many tracks as you want. Panels and track layout are not explicitly specified."
+}
+
+sub help_detail {
+    <<'HELP';
+This helps create IGV session files for manual reviewers
+HELP
+}
+
+sub execute {
+    my $self=shift;
+
+    my $output_fh = Genome::Sys->open_file_for_writing($self->output_file);
+    my $xml_writer = new XML::Writer(
+        OUTPUT => $output_fh,
+        NEWLINES => 1,
+        ENCODING => 'utf-8'
+    );
+
+    $xml_writer->xmlDecl('UTF-8', 'no');
+    $xml_writer->startTag(
+        'Session',
+        genome => $self->reference_name,
+        locus => '1:1-100',
```

Comment:
Does this tool have limitations on what references it accepts?  (What happens if the chosen reference doesn't have a chromosome `1` but instead, say, `chr1`?)